src/main/java/com/aerotrack/AerotrackCloudApp.java

```diff
@@ -5,16 +5,18 @@
 import software.amazon.awscdk.Environment;
 import software.amazon.awscdk.StackProps;
 
+import static com.aerotrack.utils.Utils.getResourceName;
+import static com.aerotrack.utils.Utils.isPersonalDeployment;
+
 public class AerotrackCloudApp {
     public static String PIPILINE_STACK = "PipelineStack";
     public static String INFRA_STACK = "InfraStack";
 
     public static void main(final String[] args) {
         App app = new App();
 
-        String devEnvironment = System.getenv("AEROTRACK_DEV");
-        if (devEnvironment != null && !devEnvironment.isEmpty()) {
-            String stackName = devEnvironment + "-" + INFRA_STACK;
+        if (isPersonalDeployment()) {
```

Comment:
questa perchè in utils? serve solo a lui, basterebbe metodo privato. mettiamo in utils le code usate più di una volta